authentication/account/service/user.go

```diff
@@ -145,8 +145,27 @@ func (s *userServiceImpl) DeactivateUser(ctx context.Context, username string) (
 		}
 
 		// revoke all user's tokens
-		return s.Services().TokenService().SetStatusForAllIdentityTokens(ctx, identity.ID, token.TOKEN_STATUS_REVOKED)
+		err = s.Services().TokenService().SetStatusForAllIdentityTokens(ctx, identity.ID, token.TOKEN_STATUS_REVOKED)
+		if err != nil {
+			return err
+		}
+
+		return nil
```

Comment:
You don't need to change this at all, do you? 
`return s.Services().TokenService().SetStatusForAllIdentityTokens(ctx, identity.ID, token.TOKEN_STATUS_REVOKED)` works fine here.

Reply:
right, I initially put the calls to WIT and Tenant in the transaction, so I needed to change this line as well, but I can revert it, now that the calls to other services is out of the transaction scope.

Reply:
fixed in c7d24bd